snippets/man_gen/hubl_custom_module_fields.json

```diff
@@ -0,0 +1,519 @@
+{
+  "Field Group":{
+    "prefix": "group.group",
+    "body": [
+      "{",
+        "\"id\": \"\",",
+        "\"name\": \"field_group\",",
+        "\"label\": \"Field group\",",
+        "\"sortable\": false,",
+        "\"required\": false,",
+        "\"locked\": false,",
+        "\"hierarchical\": false,",
+        "\"max_depth\": 0,",
+        "\"children\": [],",
+        "\"type\": \"group\",",
+        "\"default\": {}, }"              
+    ],
+    "description": "Field Group"
+  },
+  "Repeater":{
+    "prefix": "group.repeater",
+    "body":[
+      "{",
+      "  \"id\": \"\",",
+      "  \"name\": \"field_group\",",
+      "  \"label\": \"Field group\",",
+      "  \"sortable\": false,",
+      "  \"required\": false,",
+      "  \"locked\": false,",
+      "  \"occurrence\": {",
+      "    \"min\": null,",
+      "    \"max\": null,",
+      "    \"sorting_label_field\": null,",
+      "    \"default\": null",
+      "  },",
+      "  \"hierarchical\": false,",
+      "  \"max_depth\": 0,",
+      "  \"children\": [],",
+      "  \"type\": \"group\",",
+      "  \"default\": {} }"
+      
+    ],
+    "description": "Repeater Group"
+  },
+  "Date Field": {
+    "prefix": "field.date",
+    "body": [
+      "{",
+      "\"id\": \" \",",
+      "\"name\": \"${1:date_field}\",",
+      "\"label\": \"Date field\",",
+      "\"sortable\": false,",
+      "\"required\": false,",
+      "\"locked\": false,",
+      "\"hierarchical\": false,",
+      "\"max_depth\": 0,",
+      "\"step\": 1,",
+      "\"type\": \"date\",",
+      "\"default\": null }"
+    ],
+    "description": "Date field"
+  },
+  "Date Time Field": {
+    "prefix": "field.datetime",
+    "body": [
+      "{",
+      "\"id\": \"\",",
+      "\"name\": \"datetime_field\",",
+      "\"label\": \"Date and time field\",",
+      "\"sortable\": false,",
+      "\"required\": false,",
+      "\"locked\": false,",
+      "\"hierarchical\": false,",
+      "\"max_depth\": 0,",
+      "\"step\": 30,",
+      "\"type\": \"datetime\",",
+      "\"default\": null }"
+    ],
+    "description": "Date and Time Field"
+  },
+  "Link Field": {
+    "prefix": "field.link",
+    "body": [
+      "{",
+      "\"id\" : \"\",",
+      "\"name\" : \"link_field\",",
+      "\"label\" : \"Link field\",",
+      "\"sortable\" : false,",
+      "\"required\" : false,",
+      "\"locked\" : false,",
+      "\"hierarchical\" : false,",
+      "\"max_depth\" : 0,",
+      "\"supported_types\" : [ \"EXTERNAL\", \"CONTENT\", \"FILE\", \"EMAIL_ADDRESS\", \"BLOG\" ],",
+      "\"type\" : \"link\",",
+      "\"default\" : {",
+      "\"url\" : {",
+      "\"content_id\" : null,",
+      "\"type\" : \"EXTERNAL\",",
```

Comment:
Might be cool if this was something like `${1|EXTERNAL,CONTENT,FILE,EMAIL_ADDRESS,BLOG|}` so the developer could easily choose a supported type

Reply:
YES!